tlog-cosignature.md

```diff
@@ -0,0 +1,100 @@
+# Transparency Log Cosignatures
+
+A cosignature is a statement by a transparency log [witness][] that it ensured
```

Comment:
nit: /ensured/verified ?

A witness has no power to _ensure_ that the checkpoint was consistent before it was issued by the log, only to check after the fact that it is.